pennylane/__init__.py

```diff
@@ -30,7 +30,7 @@
 import pennylane.init
 import pennylane.templates
 import pennylane.qnn
-from pennylane.templates import template, broadcast
+from pennylane.templates import template
```

Comment:
If we want to display ``broadcast`` in the API of the templates module, docs complains that there are two functions of the same name, ``pennylane.broadcast`` and ``pennylane.templates.broadcast``.

I currently don't remember why we want top-level import of this function, but this would fix it.

Reply:
@josh146 would you know? I have a feeling we discussed this before...

Reply:
I decided to keep this PR free of the import debate and simply import it in the templates API, but using the ``qml.broadcast.rst`` file.